### PR TITLE
Update information about support for image and video capture

### PR DESCRIPTION
**Pull request by hamishwillee**
* [x] Update imgcap section in config file docs
* [x] Update vidcap section in config file docs
* [x] Update overview doc to remove limitation/statements image/video storage not supported
* [x] Update overview doc to state limitation that defaults for image/video cannot be overwritten in camera definition file (linking to associated issue).

### CODE REVIEW

File: en/guide/configuration_file.md [hunk 1]
```diff
@@ -109,18 +109,47 @@ Example (Ubuntu):
...
 ```
 [imgcap]
 location=~/Temp/
 ```
 
+### [vidcap] {#vidcap}
+
+Key | Description | Default
+--- | --- | ---
+`width` | TBD | TBD
```
Comment (hamishwillee) [inline en/guide/configuration_file.md]: This is waiting on https://github.com/Dronecode/csd_docs/issues/22

File: en/guide/configuration_file.md [hunk 2]
```diff
@@ -109,18 +109,50 @@ Example (Ubuntu):
...
 
 Example:
 ```
 [imgcap]
 location=~/Temp/
 ```
 
+### [vidcap] {#vidcap}
+
+<!-- These specify the default values for video capture, that will eventually be over ridable by parameter settings from MAVLink -->
```
Comment (hamishwillee) [inline en/guide/configuration_file.md]: @lbegani My understanding is that **all** vidcap and imagecap keys all define _default_ values for image/video capture that it should be possible to update via mavlink (ie QGC) once https://github.com/Dronecode/camera-streaming-daemon/issues/161 is done. Is that correct?

If so, I'll add a note about that at the top of these sections.

Comment (lbegani) [reply en/guide/configuration_file.md]: Yes, other than `location`, everything else should be settable

Comment (hamishwillee) [reply en/guide/configuration_file.md]: Thanks very much. I have updated. This is now ready for review :-)

**Comment by hamishwillee** [comment]
@lbegani I had to merge this as I've been losing track of related changes I made. Can you scan it for correctness though?

File: en/guide/configuration_file.md [hunk 3]
```diff
@@ -109,18 +109,54 @@ Example (Ubuntu):
...
-`location` | The local path with write permission where captured images will be saved (usually the system-standard "Temp" directory) | -
+`width` | Width of the image to be captured in pixels. | Full width of camera frame for sensor type (i.e 1080P - 1920, 720P - 1280, etc.)
+`height` | Height of the image to be captured in pixels. | Full height of camera frame for sensor type (i.e 1080P - 1080, 720P - 720, etc).
+`format` | Image format (number). <br>Possible values:<ul><li>2 (Jpeg/<code>IMAGE_FILE_JPEG</code>)</li></ul>Notes:<ul><li>Currently only Jpeg is <a href="https://github.com/Dronecode/camera-streaming-daemon/issues/163">supported for image capture</a>.</li></ul> | 2 (JPEG).
+`location` | The local path with write permission where captured images will be saved (usually the system-standard "Temp" directory). The path should be accessible and writeable. | /tmp/
 
 Example:
 ```
 [imgcap]
 location=~/Temp/
```
Comment (lbegani) [inline en/guide/configuration_file.md]: Example for other fields  can be added

Comment (hamishwillee) [reply en/guide/configuration_file.md]: Thanks, done! https://github.com/Dronecode/camera-manager-docs/commit/b07e19d182f0758431a19689371bf5c42f5aa66b

**Pull request merged**